### ISSUE TITLE
Play selector in Propertie not present in parts

### ISSUE THREAD

**Issue by Gai-Luron** [musescore/MuseScore#19829]
### Issue type

UI bug

### Bug description

Play selector not present in properties in part when selecting note.

### Steps to reproduce

Create score with 2 part. Open part ans select note. Go to properties, no Play selector exist

### Screenshots/Screen recordings

Conductor
![image](https://github.com/musescore/MuseScore/assets/1835871/e2c7d8eb-8c2f-4700-a8f4-52fd1cc2241b)

Part
![image](https://github.com/musescore/MuseScore/assets/1835871/52a6aafc-488b-4c71-86dd-3181a5949b6e)

### MuseScore Version

MuseScore version (64-bit): 4.2.0-232980304, revision: github-musescore-musescore-30a7714

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

OS: Windows 10 Version 2009 or later, Arch.: x86_64,

**Comment by cbjeukendrup** [comment]
I seem to recall that this is by design, but you're not the first who is confused by it / thinks it's a bug, and actually I don't see a very good reason to hide this property in part scores. However, I do think it should be a "global" property, so synced between the main score and all part scores. Playback data is namely only generated based on the main score, because generating it for each part score separately would be a bit wasteful.

**Comment by Gai-Luron** [comment]
Yes but if you work on part, it not easy to find the specified note to mute in main score with 300 Measure and 20 Parts. And yes this property must be synced between part and main score.

**Comment by Tantacrul** [comment]
@bkunda - yea, this rings a bell. It was definitely left out intentionally but I'm 90% certain the reason was due to a technical complexity. (Probably edge cases, like 'what happens if I select play in the main score / deselect it in parts?' etc.) 

I believe we decided to not have the setting for parts to avoid having to spend time of this complexity - and also because we didn't think it was particularly important leading up to the MS4 release.

**Comment by Gai-Luron** [comment]
Hello,
This is why this setting should be the same for the part and the conductor. Because in the workflow, if we want to modify this parameter in the part, we must switch back to the conductor, search for the measure using Ctrl+F and then go to the correct staff tu unckeck play. When there are 50 measures and 3 staves, it is realistic, when there are 300 measures and 20 staves, it is absurd because it is long and tedious and slows down the workflow enormously. Unless on MuseScore there is the possibility of switching to the correct measure of the conductor starting from the measure of a part.
